frontend/src/metabase/common/utils/doNotForwardProps.ts

```diff
@@ -0,0 +1,3 @@
+export const doNotForwardProps = (...propNamesToBlock: string[]) => ({
+  shouldForwardProp: (propName: string) => !propNamesToBlock.includes(propName),
```

Comment:
!nit use `Set` any time you can 🙏

Reply:
I think that's a good policy and I'm kicking myself a tiny bit for not thinking of that. Perhaps it's an over-optimization though given the typical length of the array (fewer than 5 props).